src/controllers/loginpage.js

```diff
@@ -87,7 +87,7 @@ define(["appSettings", "dom", "connectionManager", "loading", "cardStyle", "emby
         }), view.addEventListener("viewshow", function(e) {
             loading.show();
             var apiClient = getApiClient();
-            apiClient.getPublicUsers().then(function(users) {debugger;
```

Comment:
LOL

Reply:
well it happens. It's still better than finding things like if(false && some condition) and expect it to enter.